main.py

```diff
@@ -0,0 +1,67 @@
+
+import sys
+from PyQt5.QtWidgets import QApplication, QDialog
+from PyQt5.QtCore import QTimer
+import gui
+import numpy as np
+
+
+class SilGui(QDialog):
+    def __init__(self, parent=None):
+        super(SilGui, self).__init__(parent)

```

Comment:
in python 3 you can do  ```super().__init__(parent)  ```

Reply:
This file is removed for this PR, because it is not really necessary right now. It will be added later again.